ansible_runner/config/_base.py

```diff
@@ -43,7 +41,8 @@
     open_fifo_write,
     args2cmdline,
     sanitize_container_name,
-    cli_mounts
+    cli_mounts,
+    register_for_cleanup
```

Comment:
To keep diffs cleaner in the future.
```suggestion
    register_for_cleanup,
```